INSTALL.md

```diff
@@ -8,16 +8,33 @@ You must first install development and libraries to your host. Assuming Ubuntu
 (apply similar to RHEL derivatives):
 
 ```
-$ sudo apt-get update && sudo apt-get install -y build-essential libssl-dev uuid-dev
+$ sudo apt-get update && sudo apt-get install -y build-essential libssl-dev uuid-dev libgpgme-dev
 ```
 
 ## Install golang
 This is one of several ways to [install and configure golang](https://golang.org/doc/install).
 
+First, visit the [golang download page](https://golang.org/dl/) and pick a 
+package archive to download.  Copy the link address and download with `wget`.
+
+```
+$ export VERSION=1.10.2 OS=linux ARCH=amd64
+$ cd /tmp
+$ wget https://dl.google.com/go/go$VERSION.$OS-$ARCH.tar.gz
+```
+
+Then extract the archive to `/usr/local` (or use other instructions on go 
+installation page).
+
+```
+$ sudo tar -C /usr/local -xzf go$VERSION.$OS-$ARCH.tar.gz
+```
+
+Finally, set up your environment for go
+
 ```
-$ sudo apt-get update && sudo apt-get install -y golang
 $ echo 'export GOPATH=$HOME/go' >> ~/.bashrc
-$ echo 'export PATH=${PATH}:${GOPATH}/bin' >> ~/.bashrc
+$ echo 'export PATH=/usr/local/go/bin:${PATH}:${GOPATH}/bin' >> ~/.bashrc
```

Comment:
Adding in `/usr/local` ... would you set:
```
export GOROOT=/usr/local
```

In there as well?

Reply:
I'm no expert, but I think GOROOT is only necessary if you install in a custom location?
https://golang.org/doc/install

Reply:
Ahh... when using the *binary* distribution.  From the golang site:
_"The Go binary distributions  assume they will be installed in /usr/local/go"_

Reply:
OK.  So, good to go then?

Reply:
Hit that big green button!!